src/components/Identity/index.js

```diff
@@ -70,7 +70,7 @@ const createIdentity = ({ config, logger, cookieJar }) => {
       cookieJar.set(CUSTOMER_ID_HASH, customerIdsHash);
     }
     return lifecycle
-      .onBeforeEvent(event, normalizedCustomerIds, customerIdChanged)
+      .onBeforeEvent(event, {}, false) // FIXME: We shouldn't need an event.
```

Comment:
Though we shouldn't need an event on payload, Konductor currently requires one. It also has to have a `data` property on it. :/

Reply:
BTW, the reason this was forcing a request to the `interact` endpoint was because the third parameter of `onBeforeEvent` is supposed to indicate whether the event was due to a view start. Because we were passing `customerIdChanged`, which was `true`, other components like Personalization though that the event was due to a view start, and therefore requested a response.

Reply:
Yes, I tried it without event, kondcutor was throwing an error. That's why I added event and data.

Reply:
Yeah, probably I didn't notice the issue because I might have tested in an older session where `customerIdChanged` returned false :(